keras-dreambooth-sprint/README.md

```diff
@@ -0,0 +1,155 @@
+
+![](https://huggingface.co/spaces/keras-dreambooth/README/resolve/main/dreambooth.png)
+
+**Welcome to Keras Dreambooth event!** 🤗 
+
+In this guide, we will walk you through what we will do in this event.
+
+We will be training dreambooth models using KerasCV and building demos on them. 
+
+**Getting Started** 🚀 
```

Comment:
Feel free to use blurbs from https://github.com/huggingface/diffusion-models-class/blob/main/hackathon/dreambooth.ipynb or https://keras.io/examples/generative/dreambooth

I think it also make sense to use "DreamBooth" (instead of dreambooth) throughout.